.github/scripts/test_case.py

```diff
@@ -1521,39 +937,230 @@ def test_deployment_static_patch_pv():
+    result = pod.watch_for_delete(deployment.replicas)
+    if not result:
+        raise Exception("Pods of deployment {} are not delete within 5 min.".format(deployment.name))
+    LOG.info("Remove pvc {}".format(pvc.name))
+    pvc.delete()
+    LOG.info("Remove sc {}".format(pvc.name))
+    sc.delete()
+    LOG.info("Test pass.")
+    return
+
+
+def test_static_mount_image():
+    LOG.info("[test case] Deployment set mount image in PV begin..")
+    mount_image = "juicedata/mount:v1.0.0-4.8.0"
+    # deploy pv
+    pv_name = "mount-image-pv"
+    pv = PV(name=pv_name, access_mode="ReadWriteMany", volume_handle=pv_name,
+            secret_name=SECRET_NAME, parameters={"juicefs/mount-image": mount_image})
+    LOG.info("Deploy pv {}".format(pv.name))
+    pv.create()
+
+    # deploy pvc
+    pvc = PVC(name="pvc-mount-image-static", access_mode="ReadWriteMany", storage_name="", pv=pv.name)
+    LOG.info("Deploy pvc {}".format(pvc.name))
+    pvc.create()
+
+    # deploy pod
+    out_put = gen_random_string(6) + ".txt"
+    deployment = Deployment(name="app-mount-image-static", pvc=pvc.name, replicas=1, out_put=out_put)
+    LOG.info("Deploy deployment {}".format(deployment.name))
+    deployment.create()
+    pod = Pod(name="", deployment_name=deployment.name, replicas=deployment.replicas)
+    LOG.info("Watch for pods of {} for success.".format(deployment.name))
+    result = pod.watch_for_success()
+    if not result:
+        raise Exception("Pods of deployment {} are not ready within 10 min.".format(deployment.name))
+
+    # check mount point
+    LOG.info("Check mount point..")
+    volume_id = pv.get_volume_id()
+    LOG.info("Get volume_id {}".format(volume_id))
+    result = check_mount_point(out_put)
+    if not result:
+        raise Exception("mount Point of /jfs/{}/out.txt are not ready within 5 min.".format(volume_id))
+
+    LOG.info("Check mount pod image")
+    mount_pods = get_mount_pods(volume_id)
+    if len(mount_pods.items) != 1:
+        raise Exception("There should be 1 mount pods, [{}] are found.".format(len(mount_pods.items)))
+    mount_pod = mount_pods.items[0]
+    # check mount pod image
+    if mount_pod.spec.containers[0].image != mount_image:
+        raise Exception("Image of mount pod is not {}".format(mount_image))
+
+    # delete test resources
+    LOG.info("Remove deployment {}".format(deployment.name))
+    deployment.delete()
+    pod = Pod(name="", deployment_name=deployment.name, replicas=deployment.replicas)
+    LOG.info("Watch for pods of deployment {} for delete.".format(deployment.name))
+    result = pod.watch_for_delete(deployment.replicas)
+    if not result:
+        raise Exception("Pods of deployment {} are not delete within 5 min.".format(deployment.name))
+    LOG.info("Remove pvc {}".format(pvc.name))
+    pvc.delete()
+    LOG.info("Remove pv {}".format(pvc.name))
+    pv.delete()
+    LOG.info("Test pass.")
+    return
+
+
+def test_share_mount():
```

Comment:
Shall we set `pathPattern` here to cover #413?

Reply:
`pathPattern` is not the root cause in this case. `subpath` is different when using different PV and this test case has covered it.